examples/timeseries/timeseries_classification_from_scratch.py

```diff
@@ -0,0 +1,231 @@
+"""
+Title: Time series classification from scratch
+Author: [hfawaz](https://github.com/hfawaz/)
+Date created: 2020/07/21
+Last modified: 2020/07/21
+Description: Training a time series classifier from scratch on the Coffee dataset from the UCR/UEA archive.
+"""
+"""
+## Introduction
+
+This example shows how to do time series classification from scratch, starting from raw
+CSV time series files on disk, without leveraging pre-trained weights or a pre-made Keras
+Application model. We demonstrate the workflow on Coffee dataset from the [UCR/UEA
+dataset](https://www.cs.ucr.edu/%7Eeamonn/time_series_data_2018/).
+
+"""
+
+"""
+## Setup
+
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn import preprocessing
+
+"""
+## Load the data: the Coffee dataset
```

Comment:
You don't explain what the dataset is about. Please add an explanation and provide basic data exploration (how many samples for training and for validation? How many classes in the dataset? Are the classes balanced?)